sim/src/analytics.rs

```diff
@@ -180,6 +192,21 @@ impl Analytics {
             self.finished_trips.push((time, id, None, Duration::ZERO));
         }
 
+        // Trip Intersection delay
+        if let Event::TripIntersectionDelay(trip_id, turn_id, delay) = ev {
+            self.trip_intersection_delays
+                .entry(trip_id)
+                .or_insert_with(Vec::new)
+                .push((turn_id, delay));
+        }
+        //Lane Speed
```

Comment:
Syle: Spaces after `//`. I wish cargo fmt would take care of this. :(

Reply:
That is annoying. There must be a toggle somewhere? Or make an issue?

Reply:
Didn't find any options. It's mentioned in https://github.com/rust-dev-tools/fmt-rfcs/issues/17, but I'll keep looking if there's a way to enforce.